libp2p/peerstore.nim

```diff
@@ -9,20 +9,19 @@
 
 import
   std/[tables, sets, sequtils],
+  ./crypto/crypto,
   ./peerid,
   ./multiaddress
 
 type
-  ##############################
-  # Listener and handler types #
-  ##############################
+  #################
+  # Handler types #
+  #################
   
   AddrChangeHandler* = proc(peerId: PeerID, multiaddrs: HashSet[MultiAddress])
-
-  EventListener* = object
-    # Listener object with client-defined handlers for peer store events
-    addrChange*: AddrChangeHandler
-    # @TODO add handlers for other event types
+  ProtoChangeHandler* = proc(peerId: PeerID, protos: HashSet[string])
```

Comment:
Is FooChangeHandler the common name here? Nitpick, but something like FooUpdateHandler might be a tiny bit more descriptive, "Change" to me suggests something could be breaking.

Reply:
I agree. `change` was selected to stay as close as possible to [similar events and naming](https://github.com/libp2p/js-libp2p/tree/master/src/peer-store#events) in the js-implementation, but I am happy to update naming to `update` :)

Reply:
I think the naming convention makes sense here.